client.go

```diff
@@ -223,6 +223,9 @@ type ClientOptions struct {
 	MaxErrorDepth int
 	// Default event tags. These are overridden by tags set on a scope.
 	Tags map[string]string
+	// EnableLogs is a boolean flag to control if log envelopes will be generated and
+	// sent to Sentry via the logging API.
+	EnableLogs bool
```

Comment:
This should be in an experimental namespace.

Passing in a struct is probably a pain for users, so I'd be open to actually calling this `ExperimentalEnableLogs`

Reply:
The Go SDK has experienced this kind of breaking change with `ProfilesSampleRate`. I guess `EnableLogs` is fine already.

Reply:
Let's keep it as is. Given that we will be in EA soon, I want to avoid removing options a few weeks later.